index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="/CSS/style.css">
```

Comment:
- Currently your styles are not being applied to your site because you are navigating into your `css` folder wrongly, I suggest using `./CSS/style.css` . Please fix this 🙏 
- Also, all your images and background images are not displaying on your site, also because you are navigating into the `img` folder wrongly. Please start with a `./` when navigating into any folder. Ensure all images are displayed properly. Please fix this 🙏 

Your project:
![nnftnftg](https://user-images.githubusercontent.com/73607512/228502437-6c826626-d09b-4afe-ac87-f2412752a345.PNG)